### DISCUSSION TITLE
GitHub Actions for CI/CD are not recognized or run

### DISCUSSION THREAD

**Discussion by JonathanCruse** [jasontaylordev/CleanArchitecture#1122] [Q&A] [answered]
Good evening,

I am having trouble getting the CI/CD pipeline to work. I used version 8.0.3 with react template.

- Azure CLI is installed and connected
- GitHub CLI is installed and connected

What did I change?
I changed only the location for resources, it's correctly use in the Azure portal, so that cannot be the issue.

Steps:
1.  gh repo create RepoName --public --clone (✔️)
2. dotnet new ca-sln -cf React (✔️)
3. changing location in setup.ps1 (✔️)
4. run + test solution locally (✔️)
5. .\.scripts\setup.ps1 (✔️, resources are correctly created in GitGub Environments and in Azure Portal)
6. git push (❌) "fatal: The current branch master has no upstream branch." The suggested solution ("git push --set-upstream origin master") resulted in another problem: "error: failed to push some refs to 'https://github.com/[...].git'". I tried creating an initial commit from visual studio, which worked, but GitHub did not recognize the actions. 
7. Adding build.yml, cicd.yml and deploy.yml to solution file, in build.yml I also changed from on: pull_request to on: push. This resulted in build action is available in GitHub, but it has not been triggered.
![28-02-_2024_21-57-39](https://github.com/jasontaylordev/CleanArchitecture/assets/43381476/76bee8f3-9b5a-4ff1-82fe-1ce48f2d12a6)

Any ideas what I might have done wrong?

**Comment by jasontaylordev** [comment, marked as answer]
Try this after step 2:

```bash
git add .
git commit -m "New project"
git push
```

Then continue on to step 3.

**Reply by JonathanCruse** [reply]
Thanks for your quick response, that resolved my issue!
